src/common/search/FilterPlays.jsx

```diff
@@ -1,6 +1,6 @@
 import { Modal } from "common";
 import { getAllCreators, getAllLevels, getAllTags } from "meta/play-meta-util";
-import { useContext, useState } from "react";
+import { useContext, useState, useEffect } from "react";
```

Comment:
Where are you using useEffect?

Reply:
Will remove it. I added that earlier to have something which I handled in different way